### PR TITLE
[FSR] Additional info components for employment sections

### PR DESCRIPTION
**Pull request by josephrlee**
## Summary

- Added 3 additional info components to the veteran and spouse gross monthly income, deduction checklist and deduction input list to help veterans calculate their gross monthly income and deduction amounts

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#57389

## Testing done

- Developed and tested locally

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Veteran  | staging va gov_manage-va-debt_request-debt-help-form-5655_gross-monthly-income | localhost_3001_manage-va-debt_request-debt-help-form-5655_gross-monthly-income (4) |
| Spouse Gross monthly income | staging va gov_manage-va-debt_request-debt-help-form-5655_spouse_gross-monthly-income | localhost_3001_manage-va-debt_request-debt-help-form-5655_gross-monthly-income (7) |
| Veteran Deduction checklist | staging va gov_manage-va-debt_request-debt-help-form-5655_deduction_checklist | localhost_3001_manage-va-debt_request-debt-help-form-5655_gross-monthly-income (5) |
| Spouse Deduction checklist | staging va gov_manage-va-debt_request-debt-help-form-5655_deduction_checklist | localhost_3001_manage-va-debt_request-debt-help-form-5655_gross-monthly-income (8) |
| Veteran Deduction input list | staging va gov_manage-va-debt_request-debt-help-form-5655_deduction-input-list | localhost_3001_manage-va-debt_request-debt-help-form-5655_gross-monthly-income (6) |
| Spouse Deduction input list | staging va gov_manage-va-debt_request-debt-help-form-5655_spouse-deduction-input-list | localhost_3001_manage-va-debt_request-debt-help-form-5655_gross-monthly-income (9) |

## What areas of the site does it impact?

FSR veteran and spouse employment history sections

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/financial-status-report/components/householdIncome/GrossMonthlyIncomeInput.jsx [hunk 1]
```diff
@@ -151,6 +150,26 @@ const GrossMonthlyIncomeInput = props => {
             : ''
         }
       />
+      <va-additional-info
+        trigger="How to calculate gross monthly income?"
```
Comment (amponce) [inline src/applications/financial-status-report/components/householdIncome/GrossMonthlyIncomeInput.jsx:154]: Is this one a question or a statement?

File: src/applications/financial-status-report/components/householdIncome/PayrollDeductionSampleTable.jsx
Comment (amponce) [inline src/applications/financial-status-report/components/householdIncome/PayrollDeductionSampleTable.jsx:1]: Are we keeping the table after all?

File: src/applications/financial-status-report/components/householdIncome/SpouseGrossMonthlyIncomeInput.jsx [hunk 2]
```diff
@@ -151,6 +151,28 @@ const SpouseGrossMonthlyIncomeInput = props => {
             : ''
         }
       />
+      <va-additional-info
+        trigger="How to calculate gross monthly income? !"
```
Comment (amponce) [inline src/applications/financial-status-report/components/householdIncome/SpouseGrossMonthlyIncomeInput.jsx:155]: Bang sign snuck in

**Review by amponce — CHANGES_REQUESTED** [review]
It's looking good, but I spotted rouge bang ! on `src/applications/financial-status-report/components/householdIncome/SpouseGrossMonthlyIncomeInput.jsx` and I left a comment about some of the titles reading as a question vs other similar comments without the question mark, just want to verify that's correct

File: src/applications/financial-status-report/components/householdIncome/PayrollDeductionSampleTable.jsx
Comment (josephrlee) [reply src/applications/financial-status-report/components/householdIncome/PayrollDeductionSampleTable.jsx:1]: I want to. Lol, but I'm not using it. I kind of wanted to test it.

**Review by amponce — APPROVED** [review]
looks good, the only thing I noticed is the table is still in this commit, is that meant to go out?

**Pull request merged**